src/frontend/src/flows/manage/authenticatorsSection.ts

```diff
@@ -107,7 +107,13 @@ export const authenticatorsSection = ({
         <ul>
           ${authenticators.map((authenticator, index) =>
             authenticatorItem({
-              authenticator,
+              authenticator: {
+                ...authenticator,
+                // Only show rpId if cleanup is recommended
+                rpIdLabel: cleanupRecommended
```

Comment:
What if we pass `cleanUpRecommended` to `authenticatorItem` and that one decides to render the `rpIdLabel` or not depending on the value?

I think it's more consistent meaning of `rpIdLabel` than removing it just because we don't want to render it.

Reply:
Yes, `showRpId` makes more sense to me.

I didn't like passing another authenticator data than is created.